.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
since these are specific settings for the IDE, let's add .vscode/ to .gitignore file
it's good to be mindful of one's teammates and not force preferences on them unnecessarily